app/lib/services.rb

```diff
@@ -28,4 +28,19 @@ def self.rummager
       Plek.new.find('rummager'),
     )
   end
+
+  def self.rummager_enum(rummager_params, window: 100, size: Float::INFINITY)
```

Comment:
Perhaps this can be moved into gds-api-adapters? The `Services` module is for API clients itself, and it would be useful to have add this auto-paginating option available elsewhere too.

Reply:
Good Idea, I'll put up a PR